src/Cli/Microsoft.DotNet.Cli.Utils/Microsoft.DotNet.Cli.Utils.csproj

```diff
@@ -17,6 +17,7 @@
     <!-- Avoid https://github.com/dotnet/arcade/issues/9305 -->
     <AutoGenerateBindingRedirects>false</AutoGenerateBindingRedirects>
     <Nullable>enable</Nullable>
+    <NoWarn Condition="'$(TargetFramework)' == 'net472'">$(NoWarn);Nullable</NoWarn>
```

Comment:
When multi-targeting between `net472` or `netstandard2.0` and anything else, disable null checking on those TF. Otherwise you end up with non-sense warnings.

Reply:
Shouldn't we fix this in a more central location so it applies to all projects and not just cli util?

Seems like I could just chagne my logic here that's DisableNullableWarnings to a NoWarn addition: https://github.com/dotnet/sdk/pull/45420/files#diff-50e91c82311ea26f2a73202525dfdf2b0a89c178ee666b2bf3ad4c84ac4c06dc

Reply:
That is fine by me, I was just trying to demonstrate the way to approach this. Did small scale for that.